src/styles/blocks/main.scss

```diff
@@ -0,0 +1,26 @@
+.main {
+  padding: 50px 40px;
+  display: grid;
+  justify-content: center;
+  place-items: center;
+  grid-template-columns: repeat(1, 200px);
```

Comment:
Ensure that the size of the grid columns (200px) and the gap between them matches the design specifications. If the design specifies different sizes, you should update this accordingly.